chapters/annotations.tex

```diff
@@ -39,14 +39,19 @@ \section{Documentation}\label{annotations-for-documentation}\label{documentation
 The \fmtannotationindex{Documentation} annotation has the following contents, where the \lstinline!info! and \lstinline!revisions! annotations are described in \cref{annotation-info-revisions}, and the \lstinline!figures! annotation is described in \cref{annotations-for-figures}:
 \begin{lstlisting}[language=modelica]
 record Documentation
-  String info = "" "Description of the class";
-  String revisions = "" "Revision history";
+  constant String info = "" "Description of the class";
+  constant String revisions = "" "Revision history";
   Figure[:] figures = {}; "Simulation result figures";
 end Documentation;
 \end{lstlisting}
 
 How the tool interprets the information in \lstinline!Documentation! is unspecified.
 
+\subsection{How to interpret annotation definitions}
```

Comment:
Title case:
```suggestion
\subsection{How to Interpret Annotation Definitions}
```

Reply:
Also, use title which connects better with the notation section in the introduction:
```suggestion
\subsection{Record Notation for Annotation Definitions}\label{annotation-record-notation}
```

… and add a link to this section from the notation section in the introduction.